src/main/java/dev/Roach/AlertService.java

```diff
@@ -0,0 +1,85 @@
+package dev.Roach;
+
+import dev.Roach.datamodel.game.Game;
+import dev.Roach.datamodel.game.GamePojo;
+import dev.Roach.fetchers.GamesFetcher;
+import lombok.Setter;
+
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.util.List;
+
+@Setter
+public class AlertService {
+
+    private HttpClient httpClient = HttpClient.newBuilder().build();
+
+    private GamesFetcher gamesFetcher = new GamesFetcher();
+
+    public boolean createOrUpdateAlert(String email, int gameID, double price) {
+        String url = String.format("https://www.cheapshark.com/api/1.0/alerts?action=set&email=%s&gameID=%d&price=%.2f",
```

Comment:
Everything up to action= can be put into a variable to reduce the number of magic strings

Reply:
done